lib/express-middleware/authentication.js

```diff
@@ -57,15 +54,26 @@ function showNoPasswordError(res) {
   )
 }
 
+let encryptedPassword
+
 /**
  * @param {Request} req
  * @param {Response} res
  * @param {NextFunction} next
  */
 function authentication(req, res, next) {
-  if (!process.env.PROTOTYPE_PASSWORD) {
+  const password = process.env.PROTOTYPE_PASSWORD
+
+  if (!password) {
     showNoPasswordError(res)
-  } else if (
+    return
+  }
+
+  if (encryptedPassword === undefined) {
```

Comment:
I realised a bit too late last week that `encryptPassword()` already does this check